packages/variable-editor/src/protocol/ClientContextProvider.tsx

```diff
@@ -5,11 +5,11 @@ import type { Client, ClientContext } from './types';
 const defaultClientContext: any = undefined;
 const ClientContextInstance = createContext<ClientContext<any>>(defaultClientContext);
```

Comment:
still `any` here.
why can't you directly here define that this is a string context? Then the generic stuff wouldn't even be necessary...

Reply:
I defined the data that is being sent/received as a generic, as it will include different kinds of meta data for different kinds of editors and therefore be of a different type.
`export type Data<T> = { context: DataContext; data: T };`

But yes, for now it is a simple string in any case, so I could simplify it for now.

Reply:
I'm always a fan of keeping things in mind which may come later, but that doesn't mean that the code needs to show these cases already. 
IMO it is easier to make these changes when (in this case) a second editor will be implemented. Because then we know better what we really need.

Maybe we never share this context overall editors, and then we made this generic stuff here but never used it...